src/secrets.ts

```diff
@@ -56,6 +56,7 @@ export class Secrets extends Construct {
             domain: 'github.com',
             appId: '',
             personalAuthToken: '',
+            runnerLevel: 'repo',
```

Comment:
Changing the defaults will erase the secret on upgrade for existing users.

Reply:
Ok have to check this to understand it completely- but I assume it's the right place to store it...

Reply:
Got it and changed it - own secret now...